metadata-ingestion/tests/unit/utilities/test_perf_timer.py

```diff
@@ -0,0 +1,41 @@
+import time
+
+from datahub.utilities.perf_timer import PerfTimer
+
+
+def test_perf_timer_simple():
+    with PerfTimer() as timer:
+        time.sleep(1)
+        assert round(timer.elapsed_seconds()) == 1
+
+    assert round(timer.elapsed_seconds()) == 1
```

Comment:
```suggestion
    assert pytest.approx(timer.elapsed_seconds()) == 1
```
Can do the same for others

Reply:
done